webpack/assets/javascripts/react_app/components/common/forms/FormInputs.stories.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react';
+import { Row } from 'patternfly-react';
+
+import Story from '../../../../../../stories/components/Story';
+import NumericInput from './NumericInput';
+
+const NumericInputStory = props => {
+  const [value, setValue] = useState([5, '5', '5 Gb']);
+
+  return (
+    <div className="container">
+      <Row>
+        <NumericInput
+          value={value[0]}
+          label="NumericInput"
+          name="numeric"
+          onValueChange={(valueAsNumber, valueAsString, input) =>
```

Comment:
I would still use the `NumericInputs` `onChange` instead of `onValueChange` prop
as this is the prop they named this way and it looks clearer to the consumer

Reply:
And do you have any idea how to avoid the confusion about the difference on it's arguments?
`onChange` here is given value as argument instead of the event, what confuses the consumer.